ui/src/dataLoaders/components/configureStep/ConfigureDataSourceSwitcher.tsx

```diff
@@ -31,8 +31,6 @@ class ConfigureDataSourceSwitcher extends PureComponent<Props> {
             <Scraping onClickNext={onClickNext} buckets={buckets} />
           </div>
         )
-      case DataLoaderType.CSV:
```

Comment:
rip

Reply:
i cant believe that was in there for so long. oops